samples/extensions/full_screen_exclusive/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2023, Holochip Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 the "License";
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
```

Comment:
I actually took the conservative approach that, 1) I ensured that this sample runs on all platforms, 2) if it runs on a Windows platform, then it will showcase the fullscreen exclusive extension feature, otherwise 3) it simply is a "hello_triangle". 
Which, I also mentioned such, in the introduction section in its README file.

Reply:
> As this should be a windows-only sample, you might make it windows-only by adding `if (WIN32)` here.

I actually took the conservative approach that, 1) I ensured that this sample runs on all platforms, 2) if it runs on a Windows platform, then it will showcase the fullscreen exclusive extension feature, otherwise 3) it simply is a "hello_triangle".
Which, I also mentioned such, in the introduction section in its README file.

Reply:
I don't see, why it should be worth to have yet another hello_triangle sample named full_screen_exclusive for non-windows platforms.
I think, this sample doesn't give anything to non-windows users, so it shouldn't even be available on non-windows platforms.

Reply:
> ** [asuessenbach](/asuessenbach)**

Okay, I will confirm this with Steve, and update it accordingly. Meantime, all the other changes are made corresponding to your comments.